web/client/plugins/Measure.jsx

```diff
@@ -163,14 +164,24 @@ const Measure = connect(
 // the connect for mapType is needed in case the mapType is not provided by the hash pathname
 const MeasurePlugin = connect(
     createSelector([
-        mapTypeSelector
-    ], (mapType) => ({
-        mapType
+        mapTypeSelector,
+        coordsAeronauticalEnabledSelector
+    ], (
+        mapType,
+        coordsAeronauticalEnabled
+    ) => ({
+        mapType,
+        coordsAeronauticalEnabled
     }))
-)((props) => {
-    return props.mapType === MapLibraries.CESIUM
-        ? null
-        : <Measure {...props} />;
+)(({coordsAeronauticalEnabled, ...props}) => {
+    return (
+        <div className={`measure-container${coordsAeronauticalEnabled ? " bottom-offset" : ""}`}>
```

Comment:
Probably it's better top-offset from the point of view of the measure toolbar.
Another possibility is to use directly inline style